src/core/session.c

```diff
@@ -28,6 +28,12 @@ static int method_close(sd_bus_message *msg, void *data,
 
 	sd_bus_message_unref(reply);
 
+	char *exec_after = sess->screencast_instance->ctx->state->config->screencast_conf.exec_after;
```

Comment:
Can screencast_instance be NULL here?

Reply:
Off the top of my head, I have to say it might be.

More importantly, see my note in the main conversation regarding why this might not be the right place for this logic.